src/core/db/repository/member_repository.py

```diff
@@ -87,4 +88,7 @@ async def get_number_of_lombariers_by_telegram_id(self, telegram_id: int) -> int
                 )
             )
         )
-        return amount.scalars().one_or_none()
+        try:
+            return amount.scalars().one()
+        except NoResultFound:
+            return 0

```

Comment:
Не нужно использовать исключения там, где можно решить задачу без них.
```suggestion
        return amount.scalars().one_or_none() or 0
```

Reply:
Отправил новый комиит